ArmaforcesMissionBot/Modules/Signups.cs

```diff
@@ -657,6 +657,40 @@ public async Task EditMission(int missionNo)
                 }
             }
         }
+		
+        [Command("edytuj-misje")]
+        [Summary("Po podaniu nazwy misji jako parametru włączy edycje danej misji (część bez zespołów).")]
+        [ContextDMOrChannel]
+        public async Task EditMission([Remainder] string title)
+        {
+            var currentlyEditedMission = SignupsData.GetCurrentlyEditedMission(Context.User.Id);
+
+            if (currentlyEditedMission == null)
+            {
+                var missionToBeEdited = SignupsData.Missions.FirstOrDefault(x => x.Title == title);
+                if (missionToBeEdited == null)
+                {
+                    await ReplyAsync($"Nie ma misji o takiej nazwie.");
+                    return;
+                }    
+                
+                if (missionToBeEdited.Owner != Context.User.Id)
+                {
+                    await ReplyAsync($"Nie nauczyli żeby nie ruszać nie swojego?");
+                    return;
+                }
+
+                // Don't want to write another function just to copy class, and performance isn't a problem here so just serialize it and deserialize
+                var serialized = JsonConvert.SerializeObject(missionToBeEdited);
+                SignupsData.BeforeEditMissions[Context.User.Id] = JsonConvert.DeserializeObject<ArmaforcesMissionBotSharedClasses.Mission>(serialized);
+                missionToBeEdited.Editing = ArmaforcesMissionBotSharedClasses.Mission.EditEnum.Started;
```

Comment:
I guess this should be extracted somewhere because it's a copy paste from other overload. No need to extract if you decide to remove the index based mission edition.